### ISSUE TITLE
Broom icon not working 

### ISSUE THREAD

**Issue by friedger** [openintents/shoppinglist#40]
Some users report that they can't use the broom anymore.

**Comment by aperomsik** [comment]
More info would be useful. It's supposed to be greyed out if there are no items to clean. Not sure if that is confusing people or something else is wrong.

**Comment by friedger** [comment]
From users: 

"when it's in shopping mode after I click on it to cross it off and it goes to bottom of list. After that I cant clear it with the sweeper button. In order to clean up the list I have to turn on the option to clean the list by "shaking" the phone. ..... 
Yeah I've restarted app after making setting changes and also cleared cache. It's just that broom icon on top right doesn't do anything now. I touch it there is no response. Previous versions it would clear out things I've crossed off on the list. But now when I touch it nothing happens.  It's suppose clear the list if things that's crossed off right? I included screen shot." 

![screenshot_20170502-125417](https://cloud.githubusercontent.com/assets/1449049/25675560/38ae968e-303f-11e7-98d9-f98f6cb9bd51.png)
